webapp/src/components/backstage/playbook_editor/outline/outline.tsx

```diff
@@ -36,6 +36,17 @@ const Outline = ({playbook, refetch}: Props) => {
     const updatePlaybook = useUpdatePlaybook(playbook.id);
     const retrospectiveAccess = useAllowRetrospectiveAccess();
     const archived = playbook.delete_at !== 0;
+    const [checklistCollapseState, setChecklistCollapseState] = useState({} as Record<number, boolean>);
```

Comment:
nit: use generic instead of type cast
```suggestion
    const [checklistCollapseState, setChecklistCollapseState] = useState<Record<number, boolean>>({});
```